src/components/editor-page/editor-pane/editor-pane.tsx

```diff
@@ -154,7 +157,7 @@ export const EditorPane: React.FC<ScrollProps> = ({ scrollState, onScroll, onMak
         maxWidth={'100%'}
         basicSetup={true}
         className={codeMirrorClassName}
-        theme={oneDark}
+        theme={darkModeActivated ? oneDark : undefined}
```

Comment:
Didn't we already have a light editor theme? This look very bare bones.

Reply:
No one I know of. This is just the default light theme.

Reply:
Our previously writte code says this:

![image](https://user-images.githubusercontent.com/52606896/196776988-d2409031-dd80-4f7a-ab6a-196d757af03a.png)

Reply:
neat is no codemirror 6 theme

Reply:
Okay, so we need to find a good light theme for CM6. I added an issue https://github.com/hedgedoc/react-client/issues/2450